### PR TITLE
Added increment/decrement feature

### PR DESCRIPTION
**Pull request by tyloo**
Hey dude!

With this PR, we'll be able to simply increment an item without having to add the entire item to an existing item. Tell me what u think ;)

Cheers!

### CODE REVIEW

**Comment by lukepolo** [comment]
Im trying to think if the use case, how do you get the cart item ID ? If you already have it you probably already have the item hash. So why directly increase it on the item directly ?

**Comment by tyloo** [comment]
How would you increment it a cool way with `LaraCart::updateItem($itemHash, 'qty', "qty + 1 ?");` ? 

The use case here is just to increment or decrement a CartItem. You are stateless regarding the qty amount this way. I would use it with "- / +" buttons on my cart.

**Comment by lukepolo** [comment]
No id just do 

$cartItem->qty++;

**Comment by lukepolo** [comment]
updateItem is kinda useless in my mind cause id rather just do things on the item it self. I thought about removing that functionality at one point but decided against it (i dont remember the reason why)

**Comment by tyloo** [comment]
$tpe = LaraCart::getItem(session('cart'));
$tpe->qty++; // <- does this persists accross requests ?

**Comment by lukepolo** [comment]
Lara Cart i want to update this items qty = x

LaraCart::updateItem($itemHash, 'qty', 5);

**Comment by tyloo** [comment]
This I'm ok, but what about increment DIRECTLY (in one line) with an itemHash.

**Comment by lukepolo** [comment]
Fair enough. Im ok with this since the use case is solid.

File: src/LaraCart.php [hunk 1]
```diff
@@ -209,6 +209,42 @@ public function addItem(CartItem $cartItem)
...
+        $item = array_get($this->getItems(), $itemHash);
+        if ($item->qty > 1) {
+            $item->qty--;
+
+            return $item;
+        }
+
+        $this->removeItem($itemHash);
+
+        return null;
```
Comment (lukepolo) [inline src/LaraCart.php]: can you return the item ?

Comment (lukepolo) [inline src/LaraCart.php]: or just no return at all.

Comment (tyloo) [inline src/LaraCart.php]: I'd return $this->removeItem($itemHash); but removeItem doesn't return anything itself.

Other than that can you tag this release please ? So I can composer update it ;)

**Comment by tyloo** [comment]
@lukepolo : The cart is not updated across requests. Is it because `$this->update();` is not called after the operations ?

**Comment by lukepolo** [comment]
correct it should be called.

**Comment by lukepolo** [comment]
Really we can probably add update in the magic method trait so that way we dont have to worry about that.

**Pull request merged**